utilities/test_suite/CMakeLists.txt

```diff
@@ -91,159 +91,159 @@ else()
     elseif(NOT DEFINED BACKEND)
         set(BACKEND "CPU")
     endif()
-endif(NOT RPP_FOUND)
 
-# RPP installation - Audio support check
-set(RPP_AUDIO_AUGMENTATIONS_SUPPORT_FOUND 0)
-if(EXISTS ${RPP_INCLUDE_DIR}/rpp_audio_augmentations_support.h)
-    file(READ ${RPP_INCLUDE_DIR}/rpp_audio_augmentations_support.h RPP_AUDIO_AUGMENTATIONS_SUPPORT_FILE)
-    string(REGEX MATCH "RPP_AUDIO_AUGMENTATIONS_SUPPORT ([0-9]*)" _ ${RPP_AUDIO_AUGMENTATIONS_SUPPORT_FILE})
-    set(RPP_AUDIO_AUGMENTATIONS_SUPPORT_FOUND ${CMAKE_MATCH_1})
-endif()
+    # RPP installation - Audio support check
+    set(RPP_AUDIO_AUGMENTATIONS_SUPPORT_FOUND 0)
+    if(EXISTS ${RPP_INCLUDE_DIR}/rpp_audio_augmentations_support.h)
+        file(READ ${RPP_INCLUDE_DIR}/rpp_audio_augmentations_support.h RPP_AUDIO_AUGMENTATIONS_SUPPORT_FILE)
+        string(REGEX MATCH "RPP_AUDIO_AUGMENTATIONS_SUPPORT ([0-9]*)" _ ${RPP_AUDIO_AUGMENTATIONS_SUPPORT_FILE})
+        set(RPP_AUDIO_AUGMENTATIONS_SUPPORT_FOUND ${CMAKE_MATCH_1})
+    endif()
 
-# find required libraries
-set(Python3_FIND_VIRTUALENV FIRST)
-find_package(Python3 QUIET)
-find_package(NIFTI QUIET)
-find_library(libsnd_LIBS
-    NAMES sndfile libsndfile
-    PATHS ${CMAKE_SYSTEM_PREFIX_PATH} ${LIBSND_ROOT_DIR} "/usr/local"
-    PATH_SUFFIXES lib lib64)
-find_package(OpenMP QUIET)
-find_package(OpenCV QUIET)
-find_package(TurboJpeg QUIET)
-
-# find required python3-pip imports
-execute_process(
-    COMMAND ${Python3_EXECUTABLE} -m pip show pandas
-    RESULT_VARIABLE PANDAS_FOUND
-    OUTPUT_QUIET
-    ERROR_QUIET
-)
-
-if(Python3_FOUND)
-    if (PANDAS_FOUND EQUAL 0)
-        if(OpenMP_FOUND)
-
-            # HOST test set 1 - rpp_qa_tests_tensor_host_all - HOST Tensor-Image QA PASS/FAIL tests
-            if(TurboJpeg_FOUND)
-                if(OpenCV_FOUND)
-                    add_test(
-                        NAME rpp_qa_tests_tensor_host_all
-                        COMMAND ${Python3_EXECUTABLE} ${ROCM_PATH}/share/rpp/test/HOST/runTests.py --qa_mode 1 --batch_size 3
-                        WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
-                    )
+    # find required libraries
+    set(Python3_FIND_VIRTUALENV FIRST)
+    find_package(Python3 QUIET)
+    find_package(NIFTI QUIET)
+    find_library(libsnd_LIBS
+        NAMES sndfile libsndfile
+        PATHS ${CMAKE_SYSTEM_PREFIX_PATH} ${LIBSND_ROOT_DIR} "/usr/local"
+        PATH_SUFFIXES lib lib64)
+    find_package(OpenMP QUIET)
+    find_package(OpenCV QUIET)
+    find_package(TurboJpeg QUIET)
+
+    # find required python3-pip imports
+    execute_process(
+        COMMAND ${Python3_EXECUTABLE} -m pip show pandas
+        RESULT_VARIABLE PANDAS_FOUND
+        OUTPUT_QUIET
+        ERROR_QUIET
+    )
+
+    if(Python3_FOUND)
+        if (PANDAS_FOUND EQUAL 0)
+            if(OpenMP_FOUND)
+
+                # HOST test set 1 - rpp_qa_tests_tensor_host_all - HOST Tensor-Image QA PASS/FAIL tests
+                if(TurboJpeg_FOUND)
```

Comment:
Can we club these two `if` -- `turboJPEG and OpenCV` -- since these are bare minimum